common/changes/@itwin/tree-widget-react/Keep_eye_checkbox_position_static_on_horizontal_scroll_2023-05-26-11-12.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "**BREAKING** Make eye checkboxes static on horizontal scroll, decrease whitespace size between the node label and eye checkbox, `useVisibilityTreeRenderer` and `createVisibilityTreeNodeRenderer` now can take three new arguments - `levelfOffset`, `expansionToggleWidth` and `disableRootNodeCollapse`.",
```

Comment:
Let's split this comment into 3:
- **BREAKING** Trees: Make eye checkboxes static on horizontal scroll.
- Trees: Decrease whitespace size between the node label and eye checkbox.
- **BREAKING** `useVisibilityTreeRenderer` and `createVisibilityTreeNodeRenderer` now can take a "props" object of type `VisibilityTreeRendererProps`. It contains three new configuration options: `levelOffset`, `expansionToggleWidth` and `disableRootNodeCollapse`.

Reply:
fixed

Reply:
Maybe add one more for renaming `useVisibilityTreeRenderer

Reply:
...and I believe `"comment"` can only be a string rather than an array of strings